libraries/Netdump/src/NetdumpPacket.h

```diff
@@ -0,0 +1,325 @@
+/*
+    NetDump library - tcpdump-like packet logger facility
+
+    Copyright (c) 2019 Herman Reintke. All rights reserved.
+    This file is part of the esp8266 core for Arduino environment.
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License, or (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef __NETDUMP_PACKET_H
+#define __NETDUMP_PACKET_H
+
+#include <lwipopts.h>
+#include <IPAddress.h>
+#include <StreamString.h>
+#include "NetdumpIP.h"
+
+#define ETH_HDR_LEN 14
```

Comment:
consider a constexpr instead of a define.

Reply:
Done